test/test_cuda.py

```diff
@@ -3032,7 +3032,8 @@ def test_graph_rng_distributions(self):
                            # TODO: reenable multinomial tests if/when the implementation is capturable.
                            # ("multinomial", (input.clone(), size, True), {}),
                            # ("multinomial", (input.clone(), size // 2, False), {}),
-                           ("normal", (input.clone() + 1, input.clone()), {}),
+                           # TODO: reenable normal test when graph test failures are fixed
+                           # ("normal", (input.clone() + 1, input.clone()), {}),
```

Comment:
IMO it's ok if the variant where std is a device tensor is affected, we'll see if the variant errors in real graphed cases and act accordingly.

In the meantime, we should still test a variant where std is not a device tensor, so try changing the case to
```python
("normal", (input.clone() + 1, 1.0), {})
```

Reply:
That's probably a good idea. The test passes when I add that case, so I've updated this PR with it